README.adoc

```diff
@@ -307,9 +307,9 @@ The `\<<ID>>` that you receive is a unique identifier in the collection. Save th
 
 === Implementing the read operation
 
-The [hotspot=retrieve]`retrieve()` method handles the implementation of the `read` operation. The `Crew` collection is retrieved with the [hotspot=getCollectionRead]`MongoDatabase.getCollection()` method. Then, the [hotspot=find]`MongoCollection.find()` method retrieves a `FindIterable` object. This object is iterable for all the crew members documents in the collection, so each crew member document is concatenated into a String array and returned.
+The [hotspot=retrieve]`retrieve()` method handles the implementation of the read operation. The `Crew` collection is retrieved with the [hotspot=getCollectionRead]`MongoDatabase.getCollection()` method. Then, the [hotspot=find]`MongoCollection.find()` method retrieves a `FindIterable` object. This object is iterable for all the crew members documents in the collection, so each crew member document is concatenated into a String array and returned.
```

Comment:
Break up paragraph into multiple lines.

Also, not sure if describing the implementation of the iterable is needed. I think it should be fine to just say the object is iterable for all the crew member documents, and leave it at that.

Reply:
Will check with ID